jobs/V3/filter_search_sample.rb

```diff
@@ -21,25 +21,25 @@ def job_discovery_basic_keyword_search project_id:, company_name:, query:
   require "google/apis/jobs_v3"
 
   # Instantiate the client
-  jobs = Google::Apis::JobsV3
-  talent_solution_client = jobs::CloudTalentSolutionService.new
+  jobs                                 = Google::Apis::JobsV3
+  talent_solution_client               = jobs::CloudTalentSolutionService.new
   # @see
   # https://developers.google.com/identity/protocols/application-default-credentials#callingruby
   talent_solution_client.authorization = Google::Auth.get_application_default(
     "https://www.googleapis.com/auth/jobs"
   )
   # Make sure to set the request_metadata the same as the associated search request
-  request_metadata = jobs::RequestMetadata.new user_id:    "HashedUserId",
-                                               session_id: "HashedSessionId",
-                                               domain:     "www.google.com"
+  request_metadata                     = jobs::RequestMetadata.new user_id:    "HashedUserId",
+                                                                   session_id: "HashedSessionId",
+                                                                   domain:     "www.google.com"
 
   # Perform a search for analyst  related jobs
-  job_query = jobs::JobQuery.new query: query
-  job_query.company_names = [company_name] unless company_name.nil?
-  search_jobs_request = jobs::SearchJobsRequest.new request_metadata: request_metadata,
-                                                    job_query:        job_query,
-                                                    search_mode:      "JOB_SEARCH"
-  search_jobs_response = talent_solution_client.search_jobs project_id, search_jobs_request
+  job_query                            = jobs::JobQuery.new query: query
```

Comment:
An example of where I feel it's overly aggressive. Would prefer the empty line + comments separating these sections would reset the alignment